guide/C/ch_basics.xml

```diff
@@ -619,11 +619,151 @@
+		<guimenuitem>Preferences</guimenuitem></menuchoice> <guilabel>General</guilabel> 
+		<guilabel>Compress files</guilabel>. <acronym>SQL</acronym> storage is available 
+		via the <acronym>DBI</acronym> back end; use of MySQL and PostgreSQL requires 
+		installation of those <acronym>DBMS</acronym> packages as well as additional 
+		<acronym>DBI</acronym> drivers.</para>
+		
+		<note>
+		<para>Users can change the format at any time by using <menuchoice><guimenu>File</guimenu> 
+		<guimenuitem>Save As...</guimenuitem></menuchoice>. This will create a copy of 
+		the data file in the selected format.</para>
+		</note>
+		<sect3 id="basics-files-storage-comparison">
+			<title>Storage Comparison</title>
+			
+			<para>Each storage format has benefits and shortcomings that users should 
+			consider for their needs and abilities. See the Storage Comparison Table below 
+			for further details.</para>
+			
+			<para>The <acronym>XML</acronym> format is the most stable and established, 
+			and for this reason, it is recommended for most users. <acronym>SQL</acronym>
+			storage was added for the 2.4 release and has become an increasingly popular 
+			choice for users, but can require experience with <acronym>DBMS</acronym> 
+			applications.</para>
+			
+			<note>
+				<para>Use of a <acronym>SQL</acronym> back end for storage implies to many 
+				that <application>&app;</application> has fully implemented <acronym>DBMS</acronym> 
+				features, including multi-user and incremental data manipulation.  However, 
+				<application>&app;</application> does not currently implement these features, 
+				although it is a long term goal of the development team. </para>
+			</note>
+
+            <table frame="topbot" id="storage-comparison">
+              <title>Storage Comparison</title>
+
+              <tgroup align="left" cols="5">
+                <colspec colname="c1item" ></colspec>
+                <colspec colname="c2xml" ></colspec>
+                <colspec colname="c3sqlite" ></colspec>
+                <colspec colname="c4mysql" ></colspec>
+                <colspec colname="c5postgresql" ></colspec>
+
+                <thead>
+                  <row>
+                    <entry> </entry>
+                    <entry align="center">XML</entry>
+                    <entry align="center">SQLite</entry>
+                    <entry align="center">MySQL</entry>
+                    <entry align="center">PostgreSQL</entry>
+                  </row>
+                </thead>
+
+                <tbody>
+                <row>
+                  <entry >Installation</entry>
+  				<entry align="center" namest="c2xml"        > Default </entry>
+  				<entry align="center" namest="c3sqlite"     > libdbi - 1 </entry>
+  				<entry align="center" namest="c4mysql"      > libdbi </entry>
+  				<entry align="center" namest="c5postgresql" > libdbi </entry>
+                </row>
+
```

Comment:
I purposefully used the formats in the variable names because I find cryptic variable names supremely annoying. It made it easier for me to keep track of which column was set for which back end. Why use a meaningless abbreviation, when I can use something that is instantly obvious to even the casual reader of the code? And finally, why should it matter if this table uses such variables anyway?

Reply:
WRT repeating information, I disagree. I think it is better to be repetitive but unambiguous in a table such as this. I do not think merging them together is somehow easier or clearer to understand.